Document the compatability issue of auto mTLS and exlucdeInboundPort. (#8917)
istio/istio.io: content/en/faq/security/automtls-exlucde-port.md

---

> title: Does Auto mTLS work with "excludeInboundPorts" annotations?

Reviewer: ```suggestion
title: Does Auto mTLS work with the "excludeInboundPorts" annotations?
```

weight: 80
---

> No. When `traffic.sidecar.istio.io/excludeInboundPorts` is used on the server workloads, Istio still

Reviewer: ```suggestion
No. When `traffic.sidecar.istio.io/excludeInboundPorts` is used on server workloads, Istio still
```

configures the client Envoy to send mutual TLS by default. To change that, you need to configure

> a Destination Rule with mutual TLS mode `DISABLE` to make client send plain text to those ports.

Reviewer: ```suggestion
a Destination Rule with mutual TLS mode set to `DISABLE` to have clients send plain text to those ports.
```

---

> title: Does Auto mTLS work with the "excludeInboundPorts" annotations?

Reviewer 2: This title is confusing in my opinion. What does "work with" mean?

I suggest something along the lines of... Does Auto mutual TLS exclude ports set using "excludeInboundPorts" annotation?

Reviewer 3: Please also rename the file from `automtls-exlucde-port.md` to fix the typo.

Author: done.
